lib/components/molecules/Form/FieldToggle/FieldToggle.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { Field } from 'formik';
+import styles from './FieldToggle.style';
+import type { Props } from './types';
+
+import Label from '../../../atoms/Label/Label';
+import Input from '../../../atoms/Input/Input';
+
+const FieldToggle = ({
+  id,
+  name,
+  disabled,
+  label,
+  indicatorOn,
+  indicatorOff,
+  className,
+  inheritedStyles,
+  displayIndicators,
+  intialToggleState,
+  toggleCheckboxClick,
+  ...otherProps
+}: Props): Node => {
+  const [toggleOn, handleToggle] = React.useState(intialToggleState);
+
+  return (
+    <div className={className}>
+      <Label htmlFor={id} className="label-name">
+        {label}
+      </Label>
+      <Field id={id} name={name}>
+        {({ field, form }) => {
+          const { handleChange } = form;
+          return (
+            <div>
+              <Label htmlFor={id}>
+                <Input
+                  className="toggle-input"
+                  id={id}
+                  name={name}
+                  type="checkbox"
+                  checked={toggleOn}
+                  value={toggleOn}
+                  disabled={disabled}
+                  onClick={e => {
+                    handleChange(e);
+                    handleToggle(!toggleOn);
+                    toggleCheckboxClick(toggleOn);
```

Comment:
Why you are passing 2 diff values in these methods.  Create a const to represent the new value and then pass in to the callback. 

toggleCheckboxClick --> handleToggleCallback.

Also check toggleCheckboxClick to be a function else it might result into an error